db/seeds.rb

```diff
@@ -5,3 +5,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+require 'factory_bot_rails'
+
+books_list = FactoryBot::create_list(:book, 5000)
```

Comment:
We use `.` to separate class and class methods
```suggestion
books_list = FactoryBot.create_list(:book, 5000)
```